sonar-update-center-mojo/src/main/java/org/sonar/updatecenter/mojo/PluginHeader.java

```diff
@@ -143,6 +143,8 @@ public int getNbVersions() {
     return plugin.getAllReleases().size();
   }
 
+  public boolean isSupported() { return plugin.isSupported(); }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)